katcp/test/test_sensortree.py

```diff
@@ -1,10 +1,9 @@
-# Copyright 2009 National Research Foundation (South African Radio Astronomy Observatory)
+# Copyright 2010 National Research Foundation (South African Radio Astronomy Observatory)
```

Comment:
Any reason as to why you decided to change the year of the license?

Reply:
@ajoubertza suggested that it would be better to look at the history of the file, then the CP should reflect when the file was created.